### PR TITLE
Resolve #20 Render results on table

### LINKED ISSUE #20: Render Arrays

### PR DESCRIPTION
**Pull request by MariuszZalew**

### CODE REVIEW

**Comment by MariuszZalew** [comment]
- [ ] Nazewnictwo

File: src/app.js [hunk 1]
```diff
@@ -7,26 +7,11 @@ import config from "./config"
...
+(function bootstrap() {
+    console.log("Bootstrapping App")
 
-    api.load.league()
-        .then(list => attach.select.league(
-            render.select.league(list),
-            config.select.attrData.league
-        ));
+    return render.searchResult(api.searchResult());
+    // console.log(render.searchResult(api.searchResult()));
```
Comment (benedyktdryl) [inline src/app.js]: Remove comments.

File: src/app.js [hunk 2]
```diff
@@ -7,26 +7,11 @@ import config from "./config"
...
+})();
+// bootstrap();
```
Comment (benedyktdryl) [inline src/app.js]: Don't turn off bootstrapping our app.

File: src/render/select/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import league from "./league"
 import team from "./team"
 
+
```
Comment (benedyktdryl) [inline src/render/select/index.js]: This is unnecessary.

**Comment by benedyktdryl** [comment]
Fix comments, sync with master branch **after** @Txwxcc will merge his changes, squash commits and ping me back.

**Pull request merged**